java/src/main/java/com/vmware/OTelConfig.java

```diff
@@ -0,0 +1,62 @@
+package com.vmware;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author Sumit Deo (deosu@vmware.com)
+ */
+public class OTelConfig {
```

Comment:
This code has multiple sample classes in it. Do you need all these classes to make instrumentation work?

Reply:
Do you mean multiple methods? We do not need to deconstruct the configuration to have these many methods. I thought it will be readable. Please let me know if it is not, I'll update :)